packages/gatsby-theme-carbon/src/components/NextPrevious/NextPrevious.module.scss

```diff
@@ -0,0 +1,44 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~@carbon/elements/scss/type/styles';
+
+.wrapper {
+  background: $carbon--gray-90;
+  color: $carbon--white-0;
+}
+
+.link-container {
+  min-height: rem(128px);
+  display: flex;
+  .link {
+    color: $carbon--white-0;
+    padding: 1rem;
+    text-decoration: none;
+    width: 50%;
+    transition: 0.07s background ease;
```

Comment:
We should be able to use one of the fancy new duration tokens from carbon here https://www.carbondesignsystem.com/guidelines/motion/basics#duration-tokens

`$duration--fast-01` most likely 

Also, would be good to flatten this out and not nest the selectors, we had a big discussion w/ the Carbon team around this. 

```
.link-container {
...
}

.link-container .link {

}
```

Reply:
went for moderate--02 to match the IDL site

Reply:
@vpicone lets double check w/ @shixiedesign on motion token for this